.storybook/main.js

```diff
@@ -60,6 +61,17 @@ module.exports = {
             loader: 'postcss-loader',
             options: {
               sourceMap: true,
+              // @see https://github.com/csstools/postcss-preset-env/blob/master/INSTALL.md#webpack
+              ident: 'postcss',
+              plugins: () => [
+                /**
+                 * Enables Stage 2 options, by default.
+                 * This includes logical properties and values.
+                 * @see https://preset-env.cssdb.org/features#logical-properties-and-values
+                 * @see https://github.com/csstools/postcss-preset-env#stage
+                 */
+                postcssPresetEnv(),
```

Comment:
We could, alternatively, spec stage 3 or 4 and add `logical-properties-and-values` support separately.

Reply:
@Josh68 Any reason we wouldn't add this to our `postcss.config.js` instead?

Reply:
I hadn't even seen that was in our file structure. Will try that out now.

Reply:
Updated